src/test/rpc_tests.cpp

```diff
@@ -339,6 +339,29 @@ BOOST_AUTO_TEST_CASE(rpc_convert_values_generatetoaddress)
     BOOST_CHECK_EQUAL(result[2].get_int(), 9);
 }
 
+BOOST_AUTO_TEST_CASE(rpc_convert_named_values_generatetoaddress)
```

Comment:
do you want to be a hero and also test the logic `even if it is empty` from https://github.com/bitcoin/bitcoin/blob/cab94cc07489f704c4b95171b23be0e8025df794/src/rpc/client.cpp#L247

Reply:
I must admit that I don't understand your question. Sorry.

Reply:
Sorry for the confusion. What do you think about adding a test like:

```
    UniValue result;
    std::vector<std::string> valid_named_values = {
        "nblocks=10",
        "address=", // note that no address is provided by the = exists, an error should not be thrown
    };

    BOOST_CHECK_NO_THROW(result = RPCConvertNamedValues("generatetoaddress", valid_named_values));
```

Reply:
Looks good. Many thanks. Will try it out!